skorch/callbacks/logging.py

```diff
@@ -875,3 +876,103 @@ def on_epoch_end(self, net, **kwargs):
 
         for key in filter_log_keys(epoch_logs.keys(), self.keys_ignored_):
             self.experiment.log_scalar(key + self.epoch_suffix_, epoch_logs[key], epoch)
+
+
+class MlflowLogger(Callback):
+    """
+    >>> import mlflow
+    >>> net = NeuralNetClassifier(net, callbacks=[MLflowLogger])
+    >>> with mlflow.start_run():
+    ...     net.fit(X, y)
+
+    :param run: mlflow.entities.Run
+    :param client: mlflow.tracking.MlflowClient
+    """
+    def __init__(
+        self,
+        run=None,
+        client=None,
+        create_artifact=True,
+        terminate_after_train=True,
+        log_on_batch_end=False,
+        log_on_epoch_end=True,
+        keys_ignored=None,
+        batch_suffix=None,
+        epoch_suffix=None,
```

Comment:
Would it not be easier to set the default to `'_batch'` and `'_epoch'`, respectively? Then you don't need to do the checking against `None` down below.

Reply:
This is to avoid adding a suffix if only one of batch or epoch is logged.

Same as the [SacredLogger](https://github.com/skorch-dev/skorch/blob/master/skorch/callbacks/logging.py#L829).